modules/caddyhttp/serverless/testdata/echoserver/main.go

```
@@ -0,0 +1,57 @@
+		Body:    string(bodyBytes),
+	}
+
+	responseBytes, err := json.Marshal(requestInfo)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Error marshalling response: %v", err), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	w.Write(responseBytes)
```

golangci-lint (1.64.8)

Error return value of `w.Write` is not checked

(errcheck)